javatests/arcs/core/entity/HandleManagerTestBase.kt

```diff
@@ -474,8 +474,11 @@ open class HandleManagerTestBase {
     @Test
     fun collection_initialState() = testRunner {
         val handle = writeHandleManager.createCollectionHandle()
-            as ReadCollectionHandle<*>
+        assertThat(handle.size()).isEqualTo(0)
+        assertThat(handle.isEmpty()).isEqualTo(true)
         assertThat(handle.fetchAll()).isEmpty()
+        handle.clear().join()
+        handle.remove(entity1).join()
```

Comment:
I am a bit surprised by this one. If it is intended behavior maybe add a comment here to say so.

Reply:
Done